code/Report.php

```diff
@@ -324,17 +338,14 @@ public function getCMSActions()
      */
     public function getReportField()
     {
-        // TODO Remove coupling with global state
-        $params = isset($_REQUEST['filters']) ? $_REQUEST['filters'] : array();
-        $items = $this->sourceRecords($params, null, null);
```

Comment:
We could use `Injector::inst()->get(HTTPRequest::class)` instead to get the request.